CONTRIBUTING.md

```diff
@@ -1,15 +1,15 @@
 # Welcome!
 
-We're so glad you're thinking about contributing to the U.S. Forms System open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
+We're so glad you're thinking about contributing to the U.S. Forms System open source project! If you're unsure about anything, just ask anyway by submitting an issues or sending us an email.
 
 We encourage you to read this project's CONTRIBUTING policy (you are here), its [LICENSE](LICENSE.md), and its [README](README.md).
 
 ## How to contribute to U.S. Forms System
 
-- When contributing to this repository, please first discuss the change you wish to make via issue or email with the team mananaging this repository before making a change.
+- Before opening a pull request in this repository, please first discuss the change you wish to make via issue or email with the team mananaging this repository before making a change.
   - You can report bugs and/or request features via [Github Issues](https://github.com/usds/us-forms-system/issues). You can also ask the team questions and make suggestions by creating an issue. 
   - If you prefer email to issues, you can reach us at [forms@lists.usds.gov](mailto://forms@lists.usds.gov) with your questions and comments. We're always open to suggestions.
-- Create a [GitHub pull request](https://github.com/usds/us-forms-system/blob/master/.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md) with new functionality or fixing a bug.
+- To implement new functionality or fix a bug, you can open a [GitHub pull request](https://github.com/usds/us-forms-system/blob/master/.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md). However, **please make sure you have discussed these changes and have an agreed upon plan of action before embarking on any significant pull request (e.g. implementing features, refactoring code, porting to a different language), otherwise you risk spending time working on something that may not get merge into the project. You can do so by starting by always opening issues to track discussion regarding any changes you would like to implement.** 
```

Comment:
> may not get merge into the project

"merged" instead of merge

> by starting by always opening issues to track discussion

"by opening issues to track discussion"